docs/content/getting-started/hello-dagster.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Hello Dagster | Dagster
+description: Run dagster for the first time
+---
+
+# `hello dagster`
+
+This guide helps you build a simple data pipeline in dagster that downloads the top 10 HackerNews stories.
+
+1. Start by creating a file called `hello-dagster.py` and copying the following:
+
+    ```python file=/getting-started/hello-dagster/hello-dagster.py
+    # copy to a file called hello-dagster.py
+    from dagster import repository, asset, MetadataValue
+    import requests
+    import pandas as pd
+
+    # dagster helps you think about datasets, models, 
+    # and other objects that you want to exist as assets
+    # instead of worrying about tasks
+
+    @asset
+    def hackernews_topstory_ids():
+        """
+        Get up to 500 top stories from the HackerNews topstories endpoint.
+        API Docs: https://github.com/HackerNews/API#new-top-and-best-stories
+        """
+        newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
+        top_500_newstories = requests.get(newstories_url).json()
+        return top_500_newstories
+
+    # dependencies between assets can be expressed as function arguments
```

Comment:
Can we take this comment out? I think it's infer-able from looking at the code.

Reply:
I actually think it's hard to understand what's happening. I think inferring the upstream asset from the name of the arg is fairly magical and we need to include an explanation somewhere, whether it's as as comment, or commentary after the code sample.

Reply:
I'll add back a comment which I think strikes a nice balance between Sandy's concerns for code pars-ability while still calling out the magical pieces.